### PR TITLE
chore(Mangakakalot): remove presence

### PR DESCRIPTION
**Pull request by kyrie25**
**Describe the changes in this pull request:**
Remove presence as both sites has been shut down for almost a year now
image

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image

</details>

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]

**Comment by Slowlife01** [comment]
image

**Comment by kyrie25** [comment]
Wait what

**Comment by kyrie25** [comment]
image
Are you using the correct domain tho

**Comment by Slowlife01** [comment]
image

**Comment by kyrie25** [comment]
Let me try again with a VPN hang on

**Comment by kyrie25** [comment]
~~I asked Bas to double-check beforehand but maybe he didn't have the time~~ 💀 
Ayt closing this PR then
